examples/multi_deployment/main.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*****************************************
+  Folder Structure
+ *****************************************/
+module "migration-folders" {
+  source  = "terraform-google-modules/folders/google"
+  version = "~> 2.0"
+  # parent = "organizations/${var.organization_id}"
+  parent            = "folders/619261348000"
+  names             = concat(["shared"], var.environments)
+  set_roles         = true
+  per_folder_admins = var.per_folder_admins
+  all_folder_admins = var.all_folder_admins
+}
+
+/*****************************************
+  Velostrata Multi Project Deployment
+ *****************************************/
+
+module "velos-multi-project" {
+  source             = "../../modules/multi"
+  organization_id    = var.organization_id
+  billing_account    = var.billing_account
+  vpc_folder_id      = split("/", module.migration-folders.ids["shared"])[1]
+  velo_folder_id     = split("/", module.migration-folders.ids["shared"])[1]
+  prod_folder_id     = split("/", module.migration-folders.ids["prod"])[1]
```

Comment:
Instead if accepting an explicit "prod_folder_id" and "non_prod_folder" we should accept a list of "target_folders"

Reply:
The makes sense. If that is the case, do we need to make projects inside those folders? In the initial example by Karolis,  there was one project precreated and attached to the svpc.
https://github.com/terraform-google-modules/terraform-google-migrate/blob/9847b423563c57ac82aed9de5a659c49d5102cf2/multi/main.tf#L276

Reply:
We should also have a variable for "applications" or "projects" and create a project inside each folder for each app.

Reply:
@morgante as per our conversation offline I have kept these as is with hardcoded projects as we wont be able to iterate over a module. Please let me know if this needs to change.

Reply:
Let's still include the notion of a prefix and default project IDs. By default, create these projects:

- {prefix}-shared-network
- {prefix}-shared-migrate (velostrata)
- {prefix}-nonprod-core
- {prefix}-prod-core

Reply:
Done